ansible/roles_ocp_workloads/ocp4_workload_integreatly/tasks/workload.yml

```diff
@@ -3,6 +3,40 @@
   debug:
     msg: "Setting up workload for user ocp_username = {{ ocp_username }}"
 
+- name: Label worker MachineConfigPool
+  k8s:
+    api_version: machineconfiguration.openshift.io/v1
+    kind: MachineConfigPool
+    name: "{{ ocp4_workload_integreatly_machineconfigpool_name }}"
+    merge_type:
+    - strategic-merge
+    - merge
+    definition:
+      metadata:
+        labels:
+          "rhmi.io/worker-config": "{{ ocp4_workload_integreatly_machineconfigpool_label_value }}"
+
+- name: Create pod override KubeletConfig
+  k8s:
+    api_version: machineconfiguration.openshift.io/v1
+    kind: KubeletConfig
+    name: "{{ ocp4_workload_integreatly_kubeletconfig_name }}"
+    definition: "{{ lookup('template', ocp4_workload_integreatly_kubeletconfig_template) | from_yaml }}"
+
+# _get_machinepoolconfig_worker.resources[0].status.readyMachineCount should
+# either be the full amount of worker nodes, or 3, whichever is the least. this
+# should speed up the start of rhmi installations on clusters with more than 3
+# worker nodes.
+- name: Check worker MachineConfigPool has enough available nodes
+  k8s_facts:
+    api_version: machineconfiguration.openshift.io/v1
+    kind: MachineConfigPool
+    name: "{{ ocp4_workload_integreatly_machineconfigpool_name }}"
+  register: _get_machinepoolconfig_worker
+  retries: 20
+  delay: 60
+  until: _get_machinepoolconfig_worker.resources[0].status.readyMachineCount in [3, _get_machinepoolconfig_worker.resources[0].status.machineCount]
```

Comment:
we could make this exact, instead of waiting for 3 minimum. it'll just take longer on larger clusters